src/mbgl/map/map_impl.cpp

```diff
@@ -16,7 +16,8 @@ Map::Impl::Impl(Map& map_,
                 MapMode mode_,
                 ConstrainMode constrainMode_,
                 ViewportMode viewportMode_,
-                bool crossSourceCollisions_)
+                bool crossSourceCollisions_,
+                const optional<std::string> localFontFamily_)
```

Comment:
Would it be possible to pass the string (not optional) and just check it for emptiness??